content/articles/using-logistic-regression-in-pytorch-to-identify-handwritten-digits/index.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /implementing-gan-from-scratch/
+title: Implementing GANs from Scratch
+description: 
+author: bonface-ndolo
+date: 2021-11-19T00:00:00-18:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+ - url: /engineering-education/implementing-gan-from-scratch/hero.jpg
+   alt: Implementing GANs example image
+---
+
+### Introduction
+The odds ratio can be calculated using logistic regression when there are multiple explanatory variables. The approach is quite similar to multiple linear regression, except that the response variable is a binomial. An event's odds ratio is the final result of each variable's influence on the odds ratio.

```

Comment:
This is an abrupt introduction to the article. Also, when you introduce a concept, explain it. Odds ratio may not be familiar to all.